src/view/__tests__/helpers/react-testing-library.jsx

```diff
@@ -19,6 +18,8 @@ import {
 } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
+window.requestAnimationFrame = (ckb) => ckb();
```

Comment:
`requestAnimationFrame` is the function that was causing the tests to fail if the testing window is hidden behind other windows. By calling the callback immediately, the test work no matter if the testing window is visible or not. And it also shortens the tests running time with 10-15 seconds.